api/backoff/none.go

```diff
@@ -20,18 +20,21 @@
 
 package backoff
 
-import "go.uber.org/yarpc/api/backoff"
-
-// The None backoff strategy could be implemented as a trivial singleton, but
-// for brevity, is just a degenerate case of the exponential backoff.
-
-var noneOpts = exponentialOptions{
-	newRand: newRand,
-}
+import "time"
 
 // None is a shorted backoff strategy that will always produce a 0ms duration.
 // This strategy is intended to minimize arbitrary delays during tests or
 // maximize load on a benchmark.
-var None backoff.Strategy = &ExponentialStrategy{
-	opts: noneOpts,
+var None Strategy = &none{}
+
+type none struct{}
+
+// Backoff implements Strategy
```

Comment:
nit: we're all over the place in the codebase, but I'd stick to either ending Godocs with periods or not in the same file haha :-)

Reply:
+1

Godocs must be complete sentences ending with periods. A single line in the docs that doesn't end with a period is actually a header.

https://blog.golang.org/godoc-documenting-go-code